packages/testing/playwright/pages/components/RunDataPanel.ts

```diff
@@ -122,4 +122,19 @@ export class RunDataPanel {
 	async switchDisplayMode(mode: 'table' | 'ai' | 'json' | 'schema' | 'binary'): Promise<void> {
 		await this.root.getByTestId(`radio-button-${mode}`).click();
 	}
+
+	/**
+	 * Query method: Gets text content from a specific table cell
+	 * @param row Row index in tbody (0-based)
+	 * @param col Column index (0-based)
+	 * @returns Promise<string> The text content of the cell
+	 * @throws Error if no text content is found
+	 */
+	async getCellText(row: number, col: number): Promise<string> {
```

Comment:
We should keep asserts/throw logic inside the test itself it shouldn't be in the page objects.